tests/src/JIT/opt/Devirtualization/sealedclass.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+public class Base
+{
+    public virtual int GetValue(int value)
+    {
+        return 0x33;
+    }
+}
+
+public sealed class Derived : Base
+{
+    public override int GetValue(int value)
+    {
+        return value;
+    }
+}
+
+public class F
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static int TestSealedMethodInlining(Derived obj)
```

Comment:
I take it from the name of this method that you're not just expecting the `GetValue` call to be devirtualized, but also that it subsequently gets inlined?